conduit/transforms/fixmatch.py

```diff
@@ -7,7 +7,7 @@
 from torch import Tensor
 from typing_extensions import Self, override
 
-from conduit.data.datasets.utils import ImageTform, apply_image_transform
+from conduit.data.datasets.vision.utils import ImageTform, apply_image_transform
```

Comment:
I'm a bit concerned that this incorrect import wasn't noticed by the tests. I only noticed it with pyright.